docs/performance/tune-performance.md

```diff
@@ -365,3 +366,23 @@ Depending on which execution provider you're using, it may not have full support
 NCHW and NHWC are two different memory layout for 4-D tensors.
 
 Most TensorFlow operations used by a CNN support both NHWC and NCHW data format. The Tensorflow team suggests that on GPU NCHW is faster but on CPU NHWC is sometimes faster in Tensorflow. However, ONNX only supports NCHW. As a result, if the original model is in NHWC format, when the model is converted extra transposes may be added. The [tensorflow-onnx](https://github.com/onnx/tensorflow-onnx) converter does remove many of these transposes, but if this doesn't help sufficiently, consider retraining the model using NCHW.
+
+### Mitigate high latency variance
+
+On some platforms, onnxruntime may exhibit high latency variance during inferencing. This is caused by the constant cost model that onnxruntime use to parallelize tasks in the thread pool.
+The constant cost model breaks down a task with a calculated granularity, which stays constant to the end of task parallelization, sometimes this approach causes imbalanced load among threads.
+To offer an option as possible mitigation, onnxruntime provides a dynamic cost model which could be enbabled by session option:
+
+```
+sess_options.add_session_config_entry('session.dynamic_block_base', '2') #python API
+```
+
+Whenever set with a positive value, onnxruntime thread pool will parallelize internal tasks with a diminishing granularity.
```

Comment:
What value do we recommend for our users? Is the value based on the variance seen? If yes, let's mention this.

Reply:
Added -
"Per our tests, by far the best configuration for dynamic_block_base is 4, which lowers the variance while keeping the performance as good. "